releasenotes/notes/remove-deprecated-unroller-4693330708c681e0.yaml

```
@@ -0,0 +1,10 @@
+---
+upgrade:
+  - |
+    Removes the deprecated Unroller class in 'qiskit.transpiler.passes.basis'.
+    This class was deprecated in Qiskit Terra 0.45 and has been replaced by
+    `BasisTranslator`.
+
+    Note that `BasisTranslator` takes different arguments than Unroller, as it 
+    requires a `EquivalenceLibrary` object to be passed in. 
```

It would be nice to add an example snippet of how to use the translator instead of the unroller, if you'd like to add one
```suggestion
    Removes the deprecated ``Unroller`` class in :mod:`qiskit.transpiler.passes.basis`.
    This class was deprecated in Qiskit 0.45 and has been replaced by
    :class:`.BasisTranslator`.

    Note that :class:`.BasisTranslator` takes different arguments than ``Unroller``, as it 
    requires an :class:`.EquivalenceLibrary` object to be passed in. 
```

---

Thanks for the feedback and sounds good I will make those changes.